deployment_report/templates/viya_deployment_report.html.j2

```diff
@@ -136,11 +136,15 @@
     <h2>Storage</h2>
     <div>
       {% for db, info in report_data.kubernetes.dbInfo.items() %}
-      <h7>Database({{ db }})</h7>
+      <h7>Database ({{ db }})</h7>
       <table>
       {% for key, value in info.items() %}
         <tr>
-          <th>{{ key }}</th>
+          {% if key == "sslEnabled" %}
+          <th>{{ "SSL_Enabled" }}</th>
```

Comment:
I think the underscore can be removed from SSL_Enabled since it's a table header: SSL Enabled

Other than this, I think things look good.